README.md

```diff
@@ -117,8 +117,8 @@ submodule.
 Here's a look at all the git submodules that are currently tracked in
 the rocket-chip repository:
 
-* **chisel2**
-([https://github.com/ucb-bar/chisel](https://github.com/ucb-bar/chisel)):
+* **chisel3**
+([https://github.com/ucb-bar/chisel3](https://github.com/ucb-bar/chisel3)):
 At Berkeley, we write RTL in Chisel. For those who are not familiar
 with Chisel, please go take a look at
 [http://chisel.eecs.berkeley.edu](http://chisel.eecs.berkeley.edu). We
```

Comment:
> On Aug 16, 2016, at 9:20 AM, mwachs5 <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -117,8 +117,8 @@ submodule.
> >  Here's a look at all the git submodules that are currently tracked in
> >  the rocket-chip repository:
> >  
> > -* **chisel2**
> > -([https://github.com/ucb-bar/chisel](https://github.com/ucb-bar/chisel)):
> > +* **chisel3**
> > +([https://github.com/ucb-bar/chisel3](https://github.com/ucb-bar/chisel3)):
> >  At Berkeley, we write RTL in Chisel. For those who are not familiar
> >  with Chisel, please go take a look at
> >  [http://chisel.eecs.berkeley.edu](http://chisel.eecs.berkeley.edu). We
> 
> Does this link apply to chisel3? One of the main things you find at that link is the chisel-tutorial, is the chisel-tutorial now in chisel3?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

There is a Chisel3 version of the tutorials in the chisel3 branch of chisel-tutorial. Once we have an official Chisel3 release, we'll make an announcement on the web site, and that branch will become master.